lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -40,7 +40,6 @@ defmodule TrentoWeb.V1.SapSystemView do
     database
     |> Map.from_struct()
     |> Map.delete(:__meta__)
-    |> Map.delete(:database_instances)
```

Comment:
By the way, here we might want to use `add_system_replication_status_to_secondary_instance` so we put the pills properly